### PR TITLE
added xml file and BeforeMetod

### PR DESCRIPTION
**Pull request by ashchavinska**

### CODE REVIEW

File: src/test/java/com/qaprosoft/carina/demo/MobileSampleTest.java [hunk 1]
```diff
@@ -61,11 +70,11 @@ public void testWebView() {
             "message was not sent or captcha was not displayed");
     }
 
-    @Test(description = "JIRA#DEMO-0011")
+
```
Comment (vrudyk) [inline src/test/java/com/qaprosoft/carina/demo/MobileSampleTest.java]: Redundant space

File: src/test/java/com/qaprosoft/carina/demo/MobileSampleTest.java [hunk 2]
```diff
@@ -87,14 +96,12 @@ public void testUIElements() {
         Assert.assertTrue(uiElements.isOthersRadioButtonSelected(), "Others radio button was not selected!");
     }
 
-    @Test(description = "01")
+
```
Comment (vrudyk) [inline src/test/java/com/qaprosoft/carina/demo/MobileSampleTest.java]: Redundant space

File: src/test/java/com/qaprosoft/carina/demo/MobileSampleTest.java (hunk 1 above)
Comment (ashchavinska) [reply src/test/java/com/qaprosoft/carina/demo/MobileSampleTest.java]: done

File: src/test/java/com/qaprosoft/carina/demo/MobileSampleTest.java (hunk 2 above)
Comment (ashchavinska) [reply src/test/java/com/qaprosoft/carina/demo/MobileSampleTest.java]: done

**Comment by vrudyk** [comment]
ok

**Pull request merged**